bitnami/spark/templates/NOTES.txt

```diff
@@ -0,0 +1,37 @@
+
+** Please be patient while the chart is being deployed **
+
+1. Get the Spark master WebUI URL by running these commands:
+
+{{- if contains "NodePort" .Values.service.type }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ include "spark.fullname" . }})
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT
+{{- else if contains "LoadBalancer" .Values.service.type }}
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "spark.fullname" . }}'
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "spark.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+  echo http://$SERVICE_IP:{{ .Values.service.port }}
+{{- else if contains "ClusterIP" .Values.service.type }}
+  kubectl port-forward --namespace {{ .Release.Namespace }} svc/{{ template "spark.fullname" . }}-master-svc {{ default "8080" .Values.service.port }}:8080
+  echo "Visit http://127.0.0.1:{{ .Values.master.webPort }} to use your application"
+{{- end }}
+
+2. Submit an application to the cluster:
+
+To submit an application to the cluster the spark-submit script must be used. That script can be
+obtained at https://github.com/apache/spark/tree/master/bin.
+
+First, obtain the master IP, to do that the service type must be NodePort or LoadBalancer. Run the following command to obtain the master IP and submit your application:
+
+$ export MASTER_IP=$(kubectl get services | awk '/{{ include "spark.fullname" . }}/ { print $3 }')
```

Comment:
I don't think we need this, right? You already set the SERVICE_IP variable in the step above.

Reply:
It depends on the service type used. The first step gives you the WebUI port, but the IP depends, for example using `loadBalancer` it is the ingress IP that cannot be used for submit an application. I think it is clear to use `MASTER_IP` that is valid for all cases.

Reply:
I will add the explanation in case of `clusterIP` to submit an application.